main.go

```diff
@@ -124,11 +125,14 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: *metricsAddr,
-		LeaderElection:     *enableLeaderElection,
-		LeaderElectionID:   "metallb.io.metallboperator",
-		Namespace:          envParams.Namespace,
+		Scheme: scheme,
+		Metrics: metricsserver.Options{
+			BindAddress: *metricsAddr,
+		},
+
+		LeaderElection:   *enableLeaderElection,
+		LeaderElectionID: "metallb.io.metallboperator",
+		//		Namespace:        envParams.Namespace,
```

Comment:
This change is something what I need to double check ( in newer controller-runtime this field went away but did now see how was replaced)

Reply:
According to the description cache.DefaultNamespaces which replaced tha field

> DefaultNamespaces maps namespace names to cache configs. If set, only
	// the namespaces in here will be watched and it will by used to default
	// ByObject.Namespaces for all objects if that is nil.

we do not need that because we setup the cache to use that namespace